src/Jackett.Common/Indexers/Definitions/AnilibriaTop.cs

```diff
@@ -0,0 +1,139 @@
+        {
+            LoadValuesFromJson(configJson);
+            IsConfigured = false;
+
+            try
+            {
+                var results = await PerformQuery(new TorznabQuery());
+
+                if (!results.Any())
+                {
+                    throw new Exception("API unavailable or unknown error");
+                }
+
+                IsConfigured = true;
+                SaveConfig();
+            }
+            catch (Exception e)
+            {
+                throw new ExceptionWithConfigData(e.Message, configData);
+            }
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var template = Uri.EscapeDataString(query.GetQueryString());
+
+            if (query.IsTest)
+            {
+                template = "*";
+            }
+            else if (string.IsNullOrEmpty(template))
+            {
+                return releases;
+            }
+
+            var responseReleases = await RequestWithCookiesAsync(
+                $"{ApiBase}app/search/releases?query={template}", cookieOverride: string.Empty);
+            var ids = JArray.Parse(responseReleases.ContentString).Select(o => (long?)o["id"]).Where(id => id.HasValue)
+                            .Select(id => id.Value).ToList();
+            var torrentsInfo = new List<AnilibriaTorrentInfo>();
+
+            foreach (var id in ids)
+            {
+                var torrents = await RequestWithCookiesAsync(
+                    $"{ApiBase}anime/torrents/release/{id}", cookieOverride: string.Empty);
+                torrentsInfo.AddRange(
+                    JsonConvert.DeserializeObject<List<AnilibriaTorrentInfo>>(
+                        torrents.ContentString, new AnilibriaTopTorrentInfoConverter()));
+            }
+
+            releases.AddRange(
+                torrentsInfo.Select(
+                    torrentInfo => new ReleaseInfo
+                    {
+                        Guid = GetReleaseLink(torrentInfo.Alias),
+                        Title = $"{torrentInfo.NameMain} / {torrentInfo.Label}",
+                        Details = GetReleaseLink(torrentInfo.Alias),
+                        Poster = GetPosterLink(torrentInfo.PosterSrc),
+                        Year = torrentInfo.Year,
+                        Link = GetDownloadLink(torrentInfo.Hash),
+                        Size = torrentInfo.Size,
+                        Seeders = torrentInfo.Seeders,
+                        Peers = torrentInfo.Seeders + torrentInfo.Leechers,
+                        PublishDate = torrentInfo.CreatedAt,
+                        InfoHash = torrentInfo.Hash,
+                        Grabs = torrentInfo.Grabs,
+                        DownloadVolumeFactor = 0,
+                        UploadVolumeFactor = 0,
```

Comment:
```suggestion
                        UploadVolumeFactor = 1,
```
On public sites we set this to 1 to prevent the NoUpload flag being shown on results,since ULVF is not relevant for sites that don't give credit for seeding like private sites may do.